src/main/java/dev/jbang/source/Jar.java

```diff
@@ -24,7 +25,7 @@
  * extracted from the JAR file itself. So all Jars that refer to the same JAR
  * file will contain/return the exact same information.
  */
-public class JarSource implements Source {
+public class Jar implements Input {
```

Comment:
Not JarInput as its not a Jar (as the Jar api in jdk?)

Reply:
That's the name it had before, but then `SourceSet` should be `SourceSetInput` as well and I didn't like that name.

Reply:
Btw, was just thinking that "input" probably isn't a good name. When you `build()` a `SourceSet` you get a `Jar`, so in that case it's obviously not an input, it's an output. So that name might need to change. (It's basically a bunch of methods used for run and info that both a Jar and SourceSet have in common)

Reply:
I settled on `Code`... for now.